tests/nfv/hugepages_config.pm

```diff
@@ -0,0 +1,36 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+# Summary: Configures hugepages on boot time
+# Maintainer: Jose Lausuch <jalausuch@suse.com>
+
+use base "opensusebasetest";
+use strict;
+use testapi;
+use utils;
+
+sub run {
+    select_console 'root-ssh';
+
+    my $hugepages  = get_required_var('HUGEPAGES');
+    my $hugepagesz = get_required_var('HUGEPAGESZ');
+    my $grub_file  = '/boot/grub2/grub.cfg';
+    my $boot_line  = "default_hugepagesz=$hugepagesz hugepagesz=$hugepagesz hugepages=$hugepages";
+
+    record_info("Kernel boot command");
+    assert_script_run("sed -i 's/showopts/showopts " . $boot_line . "/' " . $grub_file);
```

Comment:
> > Probably we also need to run something like `make grub`, right?

Reply:
according to some instructions I found to set this up, a simple reboot is enough.. but that's why I still need to test it and it's WIP :)

Reply:
grub2-mkconfig was missing